hybrid/compartment_examples/inter_comp_call/base/README.md

```diff
@@ -0,0 +1,10 @@
+# "inter_comp_call-base" example
```

Comment:
Wouldn't this README be better placed in the root `inter_comp_call` folder, rather than `base`?

Reply:
Yep, I can merge the two READMEs.

Reply:
Refer to https://github.com/capablevms/cheri-examples/pull/76#discussion_r1015610691.